### ISSUE TITLE
Add `preserve` option

### ISSUE THREAD

**Issue by Antonio-Laguna** [valtlai/postcss-font-format-keywords#3]
Hi @valtlai !

After picking up development on postcss-preset-env I came across this issue: https://github.com/csstools/postcss-plugins/issues/116 and would love to add this plugin there. However to keep it consistent with other plugins we'd like to add `preserve` to keep both options. Is this something you would consider adding to the plugin or accepting a PR that adds this?

Using the same example but with `preserve: true` as an option:

```css
@font-face {
  src: url(file.woff2) format(woff2);
}

/* becomes */

@font-face {
  src: url(file.woff2) format(woff2);
  src: url(file.woff2) format("woff2");
}
```

**Comment by valtlai** [comment]
Thank you for the suggestion, and sorry for the late reply.

I’ll deprecate this package in favor of [@csstools’ version](/csstools/postcss-plugins/tree/main/plugins/postcss-font-format-keywords) and maybe convert this repo into a Deno version of that package.

**Comment by valtlai** [comment]
Fixed by e205efb9acc26112cd2c73dbef80411df9ab6a70
